util/graph.js

```diff
@@ -0,0 +1,77 @@
+/** Class that signifies a graph */
+class Graph {
```

Comment:
https://github.com/tair/mb4-service/pull/58/files#r1170646582
"Can you move this file into a new folder lib/datamodel? This class will only be used by the datamodel for now."